### PR TITLE
cloud: add TiFlash FAQ

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-faq.md [hunk 1]
```diff
@@ -91,4 +91,14 @@ TiDB provides TiDB Lightning and a Data Migration Tool to migrate data from MySQ
 
 ## How does TiDB protect data privacy and ensure security?
 
-Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
\ No newline at end of file
+Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+
+## Why TiDB Cloud does not back up and restore TiFlash data?
+
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:97]:
```suggestion
```

**Review by stevenayu — COMMENTED** [review]
LGTM

**Review by joey-yez — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-faq.md [hunk 2]
```diff
@@ -91,4 +91,12 @@ TiDB provides TiDB Lightning and a Data Migration Tool to migrate data from MySQ
 
 ## How does TiDB protect data privacy and ensure security?
 
-Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
\ No newline at end of file
+Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+
+## Can I import my data directly to TiFlash?
+
+No. When you import data to TiDB Cloud, the data is imported to TiKV. After the import is complete, you can use a SQL statement to specify which table to be replicated to TiFlash. Then, TiDB will create the specified replicas in TiFlash accordingly. For more information, see [Use an HTAP Cluster](/tidb-cloud/use-htap-cluster.md).
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:98]:
```suggestion
No. When you import data to TiDB Cloud, the data is imported to TiKV. After the import is complete, you can use a SQL statement to specify which table to be replicated to TiFlash. Then, TiDB will create the replicas of the specified table in TiFlash accordingly. For more information, see [Use an HTAP Cluster](/tidb-cloud/use-htap-cluster.md).
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**